src/app/features/post/post.component.html

```diff
@@ -21,7 +21,12 @@ <h2>{{ post.excerpt }}</h2>
         </span>
         <div class="post__header-authors">
           @for (author of post.authors; track $index) {
-            <blog-author [author]="author" size="lg"></blog-author>
+            <a
+              [attr.data-role]="$any(author).url ? 'button' : ''"
```

Comment:
It's already a link, so there's no need for a `data-role`
```suggestion
```

Reply:
Done